src/imitation/algorithms/preference_comparisons.py

```diff
@@ -407,12 +446,13 @@ def train(self, dataset: PreferenceDataset):
             ),
             collate_fn=preference_collate_fn,
         )
-        for fragment_pairs, preferences in dataloader:
-            self.optim.zero_grad()
-            loss = self._loss(fragment_pairs, preferences)
-            loss.backward()
-            self.optim.step()
-            self.logger.record("reward/loss", loss.item())
+        for ep in range(self.episodes):
```

Comment:
What does this have to do with episodes? This seems closer to number of epochs, as we iterate over the whole `dataloader` (and thereby `dataset`), which AFAICT might contain data from multiple (RL) episodes?

Reply:
Um, yeah, I meant epochs, it just seems that working on RL has had weird effects on my brain xD. Fixed now.